src/util/apiUtils.js

```diff
@@ -247,6 +247,10 @@ export const buildRequestArgs = externalRequestOpts => ({
 		headers['X-Meetup-Request-Flags'] = (flags || meta.flags).join(',');
 	}
 
+	if (meta.apiMetaHeaders) {
```

Comment:
what do you think about calling this `metaRequestHeaders`? Just checking [the relevant Meetup API docs](https://www.meetup.com/en-AU/meetup_api/#meta-headers) and it looks like there are a couple other headers that are also considered 'meta headers', so I think this one should be named something more specific.